src/App.css

```diff
@@ -35,18 +36,123 @@ nav ul li a {
   font-weight: bold;
 }
 
+.book-list {
+  display: flex;
+  flex-direction: column;
+  padding: 0 5%;
+}
+
 .book-card {
+  background-color: #fff;
+  margin-top: 2%;
+  list-style: none;
+  padding: 2%;
+}
+
+.card-container {
   display: grid;
-  gap: 25%;
-  margin-top: 3%;
+  grid-template-columns: 2fr 1fr 1px 1fr;
+}
+
+.btn-ul {
+  list-style: none;
+  display: flex;
+  flex-direction: row;
+  gap: 2%;
+  margin-top: 15px;
+}
+
+.btn-ul li:not(:first-child) {
+  border-left: 1px solid rgb(238, 221, 221);
+  padding-left: 9px;
+}
+
+.book-btn {
+  background-color: #fff;
+  border: none;
+  outline: none;
+  color: rgb(131, 14, 240);
```

Comment:
- [ ] Kindly use the given colors for the app as this is a requirement.

Your design
![image](https://github.com/MohammadYaser/book-store/assets/81932478/234f655f-55df-4b67-a28b-9badcbe2805c)

Given design
![image](https://github.com/MohammadYaser/book-store/assets/81932478/9fca68a8-804d-4253-a0a5-d630a2cd57f9)